app/views/homes/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= render "homes/about" %>
+<div class="product">
+	<div class="container">
```

Comment:
check indent file ni em nhé
setting trong VS Code ấy em, ở bottom-right: `Spaces -> Indent using spaces -> chọn 4 `, `Spaces -> Convert indentation to spaces`